### DISCUSSION TITLE
Helix doesn't fill the terminal when it is large

### DISCUSSION THREAD

**Discussion by dllu** [helix-editor/helix#4283] [General]
When I maximize the window it results in a huge gap on the bottom and right

![image](https://user-images.githubusercontent.com/14482624/195963980-47e3be37-a7b9-4707-bf5d-5b1d40f0352a.png)

However, when the window is small, it fills the available space appropriately.

![image](https://user-images.githubusercontent.com/14482624/195963975-bc996065-5f61-45a6-ba6c-989d6be261a7.png)

The terminal I'm using is Alacritty.

Other editors such as neovim do not have this issue. Here is the screenshot using neovim:

![image](https://user-images.githubusercontent.com/14482624/195964063-b0239389-efa3-402a-8bf0-3df7a1544844.png)

I'm using this on Ubuntu 20.04 with the i3 window manager.

![image](https://user-images.githubusercontent.com/14482624/195964105-a2ce763d-ff8d-437b-ac3d-a8a15017df26.png)

both `tput lines` and `stty size` show the correct dimensions of the terminal so I don't know why Helix is truncating it to only 135 lines.

I saw this related thread: https://github.com/helix-editor/helix/discussions/4236 but it does not seem to be the same issue that I'm experiencing.

**Comment by pathwave** [comment]
When you maximize the window does the helix part of it match the same size as helix was before you maximized? Maybe i3 does not send a resize event. I dont get huge gaps with bspwm, only small ones

**Reply by dllu** [reply]
It does not match the size from before resizing. It gets bigger but doesn't fill up the whole terminal.

**Comment by A-Walrus** [comment]
I was able to reproduce on `alacritty`, `kitty`, and `Gnome Console`, with Gnome as DE
